### DISCUSSION TITLE
NuGet Package Load Error

### DISCUSSION THREAD

**Discussion by jstar-z** [RehanSaeed/Serilog.Exceptions#26] [Q&A]
Get the following error when trying to install the NuGet package:

Install-Package : Unable to find package 'Serilog.Exceptions' at source ''.
At line:1 char:1
+ Install-Package Serilog.Exceptions
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Install-Package], Exception
    + FullyQualifiedErrorId : NuGetCmdletUnhandledException,NuGet.PackageManagement.PowerShellCmdlets.InstallPackageCommand

**Comment by RehanSaeed** [comment]
I've not had any trouble. Whatever problem you have, it sounds like a NuGet problem. Sorry I can't be more help, the error does not tell me much.
